apps/web/src/routes/+layout.svelte

```diff
@@ -1,25 +1,75 @@
-<script>
+<script lang="ts">
 	import Sidebar from '$lib/components/sidebar/Sidebar.svelte';
 	import { fly } from 'svelte/transition';
 	import '../app.css';
 	import toast, { Toaster } from 'svelte-french-toast';
 	import Scans from '$lib/components/scans/Scans.svelte';
 	import { browser } from '$app/environment';
+	import { Focus } from 'lucide-svelte';
+	import { onMount } from 'svelte';
+	import { api } from '$lib/api/client';
+
+	let Device: any;
```

Comment:
Je tady nutný to any?

Reply:
Koukám že to je pro `svelte-device-info`, ale to je docela divnej package, hlavně protože nemá co dělat se Sveltem. Možná bych použil [spíš tohle](https://github.com/etienne-martin/device-detector-js), kterej by měl fungovat i ne na serveru protože to používá User-agent, který můžeš získat přes [sveltekit hooky](https://kit.svelte.dev/docs/web-standards#fetch-apis-headers) nebo prostě [javascriptem](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgent) a vypnul bych SSR

Reply:
Ok hele nechám to na jindy